bucket/universal-gcode-sender.json

```diff
@@ -2,7 +2,7 @@
     "version": "2.0.7",
     "description": "Full featured gcode platform used for interfacing with advanced CNC controllers.",
     "homepage": "https://winder.github.io/ugs_website/",
-    "license": "GPL-3.0",
+    "license": "GPL-3.0-or-later",
```

Comment:
Github licenses are not explicitly `or-later`. Unless you have explicit proof on authors website it is later, you cannot use it.

Reply:
Alright, good to know.